drivers/biometrics/Kconfig.as608

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2025 Siratul Islam
+
+config BIOMETRICS_AS608
+	bool "AS608 Optical Fingerprint Sensor"
+	default y
+	depends on DT_HAS_SYNOCHIP_AS608_ENABLED
+	select SERIAL
+	select UART_INTERRUPT_DRIVEN
+	help
+	  Enable driver for AS608 optical fingerprint sensor.
+
+if BIOMETRICS_AS608
+
+config AS608_PASSWORD
```

Comment:
This should probably be in Devicetree since the password is unique to each sensor?